src/OpenTelemetry/Metrics/Tags.cs

```diff
@@ -27,6 +27,39 @@ public Tags(KeyValuePair<string, object?>[] keyValuePairs)
 
     public static bool operator !=(Tags tag1, Tags tag2) => !tag1.Equals(tag2);
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static int ComputeHashCode(KeyValuePair<string, object?>[] keyValuePairs)
```

Comment:
Don't think this change is required.

Reply:
Made sure the test is not dependent and reverted the change.